packages/shared/providers.yaml

```diff
@@ -4665,6 +4665,22 @@ typeform:
         base_url: https://api.typeform.com
     docs: https://docs.nango.dev/integrations/all/typeform
 
+typefully:
+    display_name: Typefully
+    categories:
+        - twitter
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.typefully.com
+        headers:
+            X-API-Key: Bearer ${apiKey}
+    docs: https://docs.nango.dev/integrations/all/typefully
```

Comment:
It might be good to also test the implementation using the `proxy.verification` field. A good example would be [Ashby](https://github.com/NangoHQ/nango/blob/b102a397f675fd1133b66a5c816ba1acee6c8206/packages/shared/providers.yaml#L271).

Reply:
Already mentioned by @bodinsamuel  😊